hoomd/md/methods/methods.py

```diff
@@ -227,36 +215,35 @@ class NPT(Method):
     - ``'yz`'`` (*Ly* and *Lz* are coupled)
     - ``'xyz`'`` (*Lx*, *Ly*, and *Lz* are coupled)
 
-    Degrees of freedom of the box specify which lengths and tilt factors of the
+    The degrees of freedom of the box set which lengths and tilt factors of the
     box should be updated, and how particle coordinates and velocities should be
     rescaled. The ``box_dof`` tuple controls the way the box is rescaled and
-    updated. The first three elements ``box_dof[:3]`` controls whether the x, y,
-    and z box lengths are rescaled and updated, respectively. The last three
-    entries ``box_dof[3:]`` control the rescaling or the tilt factors xy, xz,
-    and yz. All options also appropriately rescale particle coordinates and
-    velocities.
+    updated. The first three elements ``box_dof[:3]`` controls whether the *x*,
+    *y*, and *z* box lengths are rescaled and updated, respectively. The last
+    three entries ``box_dof[3:]`` control the rescaling or the tilt factors
+    *xy*, *xz*, and *yz*. All options also appropriately rescale particle
+    coordinates and velocities.
 
-    By default, the x, y, and z degrees of freedom are updated.
+    By default, the *x*, *y*, and *z* degrees of freedom are updated.
     ``[True,True,True,False,False,False]``
 
     Note:
-        If any of the diagonal x, y, z degrees of freedom is not being
+        If any of the diagonal *x*, *y*, *z* degrees of freedom is not being
         integrated, pressure tensor components along that direction are not
         considered for the remaining degrees of freedom.
 
     For example:
 
-    - Specifying all couplings and x, y, and z degrees of freedom amounts to
+    - Setting all couplings and *x*, *y*, and *z* degrees of freedom amounts to
       cubic symmetry (default)
-    - Specifying xy couplings and x, y, and z degrees of freedom amounts to
+    - Setting *xy* coupling and *x*, *y*, and *z* degrees of freedom amounts to
       tetragonal symmetry.
-    - Specifying no couplings and all degrees of freedom amounts to a fully
+    - Setting no couplings and all degrees of freedom amounts to a fully
       deformable triclinic unit cell
 
-    `NPT` numerically integrates the equations of motion using the symplectic
-    Martyna-Tobias-Klein equations of motion for NPT. For optimal stability, the
-    update equations leave the phase-space measure invariant and are manifestly
-    time-reversible.
+    `ConstantPressure` numerically integrates the equations of motion using the
+    symplectic Martyna-Tobias-Klein integrator with (TODO: document Langevin
```

Comment:
@MartinGirard please add details to the documentation on the method you implemented, with a link to a paper if applicable.

Reply:
I've added details. Let me know if you think it should be further expanded upon.

Reply:
Thanks!